src/librustdoc/clean/inline.rs

```diff
@@ -124,16 +124,8 @@ crate fn try_inline(
     let attrs = merge_attrs(cx, Some(parent_module), target_attrs, attrs_clone);
 
     cx.renderinfo.borrow_mut().inlined.insert(did);
-    ret.push(clean::Item {
-        source: cx.tcx.def_span(did).clean(cx),
-        name: Some(name.clean(cx)),
-        attrs,
-        inner,
-        visibility: clean::Public,
-        stability: cx.tcx.lookup_stability(did).cloned(),
-        deprecation: cx.tcx.lookup_deprecation(did).clean(cx),
-        def_id: did,
-    });
+    let what_rustc_thinks = clean::Item::from_def_id_and_parts(did, Some(name), inner, cx);
+    ret.push(clean::Item { attrs, visibility: clean::Public, ..what_rustc_thinks });
```

Comment:
Why does rustdoc override visibility here?

Reply:
Oops, this is left over from trying to fix the `enum E { pub V }` bug. I reverted it.